js-pkg/driftdb/src/index.ts

```diff
@@ -29,7 +30,7 @@ export class SubscriptionManager<T> {
     subscriptions: Map<string, EventListener<T>> = new Map()
 
     subscribe(subject: Key, listener: (event: T) => void) {
-        const key = JSON.stringify(subject)
+        const key = subject
```

Comment:
Maybe just rename the `subject` arg to `key` like you did in `dispatch()`?

Reply:
good catch